tests/conftest.py

```diff
@@ -53,6 +53,11 @@ def secret_factory_class(request):
     return secret_factory_fixture(request)
 
 
+@pytest.fixture(scope='session')
+def secret_factory_session(request):
+    return secret_factory_fixture(request)
+
```

Comment:
@fbalak @petr-balogh are we not going to use the default Secret, SC and Blockpool for all IO before and after upgrade ? 
If yes & we agree to that, then do we really need these **session fixtures** , as we would anyways not delete default resources? Only PVC deletions would suffice

Reply:
Default storage class is now used. What default secret should be used? Shouldn't customer create a new project with secret for her/his pods?

Reply:
@fbalak  If we are using the default SC, then we dont need to worry about the secret. The secret is already created and specified in the default RBD and CEPHFS SCs.

Reply:
Ok